### PR TITLE
Refactored models and influx by adding commonly used functions.

### PR DESCRIPTION
**Pull request by Stanbroek**
### Types of changes
- [X] refactored models
- [X] refactored influx

### CODE REVIEW

**Comment by Stanbroek** [comment]
```
============================== test session starts ===============================
platform linux -- Python 3.6.8, pytest-4.6.3, py-1.8.0, pluggy-0.12.0
rootdir: /home/stan/PSE/PSE-WEB
plugins: cov-2.7.1
collected 9 items                                                                

app/tests/test_utils_influx.py .                                           [ 11%]
app/tests/test_utils_models.py .....                                       [ 66%]
app/tests/test_utils_spotify.py .                                          [ 77%]
app/tests/test_views.py ..                                                 [100%]

----------- coverage: platform linux, python 3.6.8-final-0 -----------
Name                              Stmts   Miss  Cover
-----------------------------------------------------
app/API/__init__.py                  14      0   100%
app/API/playlist_calls.py             7      1    86%
app/API/song_calls.py                11      4    64%
app/API/track_calls.py              103     69    33%
app/API/user_calls.py                58     39    33%
app/__init__.py                      14      0   100%
app/tests/__init__.py                 0      0   100%
app/tests/presets.py                 32      4    88%
app/tests/test_utils_influx.py        8      0   100%
app/tests/test_utils_models.py       36      0   100%
app/tests/test_utils_spotify.py      11      3    73%
app/tests/test_views.py              17      0   100%
app/utils/__init__.py                 0      0   100%
app/utils/exceptions.py              13     11    15%
app/utils/influx.py                  29      7    76%
app/utils/models.py                 112      4    96%
app/utils/recommendations.py         35     24    31%
app/utils/spotify.py                 34     12    65%
app/utils/tasks.py                  128    111    13%
app/views.py                         45     25    44%
-----------------------------------------------------
TOTAL                               707    314    56%


============================ 9 passed in 2.99 seconds ============================
```

File: update_moods_worker.py [hunk 1]
```diff
@@ -7,7 +7,7 @@ def main():
     duration = sys.argv[1]
     # We Limit the traceback to keep the log files clear.
     sys.tracebacklimit = 0
-    userids = User.get_all_users()
+    userids = User.get_all_userids()
```
Comment (Diallo) [inline update_moods_worker.py]: I still believe we should use user_ids instead of userids but okay..

File: app/utils/tasks.py [hunk 2]
```diff
@@ -142,7 +142,7 @@ def update_user_tracks(access_token):
     if tracks:
         update_songmoods(tracks_features)
         client.write_points(tracks)
-        print(f"[{current_time}] Succesfully stored the data for '{user_data['display_name']}'")
+        print(f"[{current_time}] Successfully stored the data for '{user_data['display_name']}'")
```
Comment (Diallo) [inline app/utils/tasks.py]: When are we removing these print statements?

File: app/utils/models.py [hunk 3]
```diff
@@ -32,13 +32,17 @@ def create_if_not_exist(json_info, refresh_token):
...
+        return User.query.filter_by(userid=userid).first()
 
     @staticmethod
-    def get_all_users():
+    def get_all_userids():
         return [r.userid for r in db.session.query(User.userid)]
 
+    @staticmethod
+    def get_all_tokes():
+        return [r.refresh_token for r in db.session.query(User.refresh_token)]
```
Comment (Diallo) [inline app/utils/models.py]: I believe this can be a one - liner without list comprehension.

File: app/utils/tasks.py (hunk 2 above)
Comment (Stanbroek) [reply app/utils/tasks.py]: These print, in particular, are for debugging the workers.
But I do agree that there are a lot of unnecessary print statements we should cleanup.

File: app/utils/models.py (hunk 3 above)
Comment (Stanbroek) [reply app/utils/models.py]: The only thing I could find about this was:
`When passing in ORM-instrumented descriptors such as a column, each result is a named tuple, even for just one column.`
from https://stackoverflow.com/a/48467116,
witch, you could unpack using https://stackoverflow.com/a/10636583, but this just makes it less readable.

**Pull request merged**